modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -432,12 +437,6 @@ impl MintRecoveryState {
                     (out_point, output.amount, issuance_request),
                 );
             } else {
-                // put it back, incorrect amount
-                self.pending_nonces.insert(
-                    output.blind_nonce.0.into(),
-                    (issuance_request, note_idx, pending_amount),
-                );
-
```

Comment:
Happy to be corrected, but I don't see under which circumstance this would do anything